src/codecs/load_aiff.c

```diff
@@ -29,6 +29,7 @@
 */
 
 #include "SDL_endian.h"
+
```

Comment:
please revert the changes to all `*.c` files: it _should_ build fine. If not, let's see details why.